constants/mappings.ts

```diff
@@ -24,19 +24,19 @@ export const FeatureIconMap: Record<ContractType, StaticImageData> = {
 } as const;
 
 export const UrlMap: Record<ContractType, string> = {
-  "nft-drop": "nft-drop",
-  "nft-collection": "nft-collection",
-  "edition-drop": "edition-drop",
-  edition: "edition",
-  token: "token",
-  vote: "vote",
-  marketplace: "marketplace",
+  "nft-drop": "/nft-drop",
+  "nft-collection": "",
+  "edition-drop": "/edition-drop",
+  edition: "",
+  token: "",
+  vote: "/vote",
+  marketplace: "/marketplace",
   pack: "",
-  split: "split",
-  "token-drop": "token-drop",
+  split: "/split",
+  "token-drop": "/token-drop",
   // TODO (byoc)
   custom: "",
-  "signature-drop": "signature-drop",
+  "signature-drop": "/signature-drop",
   multiwrap: "",
 };
```

Comment:
this change scares me, I would love to leave this all as is and fix it in the one spot where it causes problems

Reply:
scares me because I don't have a good overview of where else this might be used if that makes sense